src/controllers/update/overseas.entity.change.controller.ts

```diff
@@ -73,27 +73,19 @@ export const resetChangeData = async (req: Request, appData: ApplicationData) =>
     const companyProfile = await getCompanyProfile(req, appData.entity_number as string) as CompanyProfile;
     reloadOE(appData, appData.entity_number as string, companyProfile);
   }
-  if (appData.update){
+  if (appData.update) {
     appData.update.registrable_beneficial_owner = undefined;
     appData.update.bo_mo_data_fetched = false;
-    await existingBoMoForNoChange(req, appData);
+    await retrieveBoAndMoData(req, appData);
   }
   return appData;
 };
 
-export const resetNoChangeData = (appData: ApplicationData) => {
+export const resetDataForChange = (appData: ApplicationData) => {
   if (appData.update){
     appData.update.registrable_beneficial_owner = undefined;
   }
   appData.beneficial_owners_statement = undefined;
   appData.payment = undefined;
   return appData;
 };
-
-const existingBoMoForNoChange = async (req: Request, appData: ApplicationData) => {
```

Comment:
I removed this function as I noticed this was a duplicate of `retrieveBoAndMoData(req, appData)`. Also as part of the `retrieveBoAndMoData(req, appData)` function, we're resetting the arrays of BO/MO data in `appData.update.review_xyz`

Reply:
Thanks for clarifying.